undertow/src/main/java/org/wildfly/extension/undertow/Host.java

```diff
@@ -66,6 +66,7 @@
     private final Set<Deployment> deployments = new CopyOnWriteArraySet<>();
     private final Map<String, AuthenticationMechanism> additionalAuthenticationMechanisms = new ConcurrentHashMap<>();
     private final HostRootHandler hostRootHandler = new HostRootHandler();
+    private volatile boolean ssoRegistered;
```

Comment:
Is this necessary?  Why not just look to see if the SSO authentication mechanism is in the above map?